main.go

```diff
@@ -47,14 +47,24 @@ func runMain() error {
 		return err
 	}
 
-	camera := lepton3.New(conf.SPISpeed)
-	err = camera.Open()
-	if err != nil {
-		return err
+	for {
+		camera := lepton3.New(conf.SPISpeed)
+		err = camera.Open()
+		if err != nil {
+			return err
+		}
+		defer camera.Close()
+		camera.SetLogFunc(func(t string) { log.Printf(t) })
+
+		runRecordings(conf, camera)
```

Comment:
The error returned by `runRecordings` shouldn't be ignored. If the error is caused by anything other than NextFrame failing (probably disk filling up or some other file handling error) then that error should be bubbled up (i.e. runMain should return it and the recorder should exit).

It order to identify if the error is caused by NextFrame, define a `nextFrameErr` type that implements the error interface and wrap the error returned by NextFrame before returning from runRecordings. 

Error type:

```go
type nextFrameErr struct {
   cause error
}

func (e *nextFrameErr) Error() string {
	return e.cause.Error()
}
```

Wrapping the error:

```go
    return nextFrameErr{err}
```


Testing for the error using a type assertion:

```go
err := runRecordings()
if err != nil {
    if _, isNextFrameErr := err.(nextFrameErr); !isNextFrameErr {
        return err
    }
}
// power cycle the camera etc...
```